contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -27,6 +28,8 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
   uint256 internal constant PPB_BASE = 1_000_000_000;
   uint64 internal constant UINT64_MAX = 2**64 - 1;
   uint96 internal constant LINK_TOTAL_SUPPLY = 1e27;
+  UpkeepFormat internal constant upkeepTranscoderVersionBase = UpkeepFormat.V1;
```

Comment:
should a constant be all caps like the above?

Reply:
done